### ISSUE TITLE
Can not work with json field

### ISSUE THREAD

**Issue by bolechen** [dcasia/conditional-container#21]
Hello. In my project I use json field in mysql for store user meta data,  the user model like this

```php
...
    /**
     * The attributes that should be cast to native types.
     *
     * @var array
     */
    protected $casts = [
        'email_verified_at' => 'datetime',
        'meta' => 'array',
    ];
```

When I use nova, i can use meta field like this, it works perfect.

```php
            Toggle::make('Have Meta', 'meta->flag'),
            Text::make('Meta Point', 'meta->point'),
            Number::make('Meta Amount', 'meta->amount'),
```

But when i use this package the field content is always empty to get from database.

```php
            Toggle::make('Have Meta', 'meta->flag'),

            ConditionalContainer::make([
                Text::make('Meta Point', 'meta->point'),
                Number::make('Meta Amount', 'meta->amount'),
            ])->if('meta->flag = true'),
```

I take a screenshot below, the top two fields is in the  `ConditionalContainer` call, is empty, the above two is normal nova fields.

Please help me, where am i wrong?

![image](https://user-images.githubusercontent.com/195015/71445908-e9b85b00-2758-11ea-845c-94e27318321a.png)

**Comment by milewski** [comment]
That`s a bit tricky to fix because the actual name of the field on the dom is `flag` not `meta->flag`, I faced the same issue previously so I built another package to work with JSON:

https://github.com/dcasia/nova-json-wrapper

```php
JsonWrapper::make('meta', [

    Toggle::make('Have Meta', 'flag'),
    Text::make('Meta Point', 'point'),
    Number::make('Meta Amount', 'amount'),

    ConditionalContainer::make([
        Text::make('Meta Point', 'point'),
        Number::make('Meta Amount', 'amount'),
    ])->if('flag = true'),

])
```

So this should work!..

**Comment by bolechen** [comment]
thx @milewski  for quickly response.

I use the `Json wrapper` package like this

```php
use DigitalCreative\ConditionalContainer\ConditionalContainer;
use DigitalCreative\ConditionalContainer\HasConditionalContainer;
use DigitalCreative\JsonWrapper\JsonWrapper;
use DigitalCreative\JsonWrapper\HasJsonWrapper;

class User extends Resource
{
    use HasConditionalContainer;
    use HasJsonWrapper;
```

but it toke me some errors:

```
Symfony\Component\Debug\Exception\FatalErrorException
Trait method availablePanelsForDetail has not been applied, because there are collisions with other trait methods on App\Nova\User
```

**Comment by milewski** [comment]
@bolechen see this for a fix https://github.com/dcasia/nova-json-wrapper/issues/4

**Comment by bolechen** [comment]
> @bolechen see this for a fix [dcasia/nova-json-wrapper#4](https://github.com/dcasia/nova-json-wrapper/issues/4) both of these package define the same method on their trait, so there is a collision you need to use class inhiretence to solve it

thx,  I will follow up on this issue to  `dcasia/nova-json-wrapper`
